.github/workflows/cypress.yml

```diff
@@ -64,7 +37,7 @@ jobs:
           NPM_CONFIG_PRODUCTION: false
         working-directory: ./my_project/client
         run: npm install
-      - name: Run against ${{ needs.Setup.outputs.BASE_URL }}
+      - name: Run against ${{ github.event.deployment_status.environment_url }}
```

Comment:
Removed the `jwalton/gh-find-current-pr@v1` action dependency as we no longer need to "figure out" what the URL is for the deployment. Heroku gives that to as when it creates the status event.

Reply:
This is why the `Setup` step above is now a LOT simpler